api/v1alpha1/status.go

```diff
@@ -120,6 +120,10 @@ type RuntimeStatus struct {
 
 	// APIGatewayStatus represents rest api gateway status
 	APIGatewayStatus *APIGatewayStatus `json:"apiGateway,omitempty"`
+
+	// MountTime represents time last mount happened 
+	// if Mounttime is early than master starting time, remount will be required
+	MountTime metav1.Time `json:"mountTime,omitempty"`
```

Comment:
Does it cause problems if the time of the machines in the cluster is inconsistent?

Reply:
Before remount there will be a check if any path is unmounted.
If no, there will not be any operation and mount time will be refreshed.

So no problems will be caused.